.emacs.d/init.el

```diff
@@ -56,15 +56,32 @@ If AUTOLOADS is non-nil, update the autoloads for that directory."
   (add-to-list 'load-path default-directory)
   (normal-top-level-add-to-load-path kotct/hub-list))
 
-;; If byte-compiled files are older, load newer version.
-(let ((load-prefer-newer t))
-  ;; require all hubs
-  (mapc (lambda (hub)
-          (require (intern (concat hub "-hub"))))
-        kotct/hub-list))
-
-;; Load the loaddefs.
-(require 'kotct-loaddefs)
+(defun kotct/load-hubs (&optional frame)
+  "Does the majority of initialization for dot/.emacs,
+by loading the hubs defined in `kotct/hub-list'.
```

Comment:
ok. doesn't really matter that much. We can always go back and work on documentation and stuff later.